gno.land/pkg/gnoweb/markdown/ext_test.go

```diff
@@ -27,7 +27,7 @@ func testGoldmarkOutput(t *testing.T, nameIn string, input []byte) (string, []by
 	require.Greater(t, len(name), 0, "txtar file name cannot be empty")
 
 	m := goldmark.New()
-	GnoExtension.Extend(m)
+	NewGnoExtension("gno.land", "r/test").Extend(m)
```

Comment:
The extension cannot be dependent on the path at initialization as it is something dynamic; ideally, we want to pass the current URL as context when parsing.
Im not even sure we should pass the domain, at last for now.